__tests__/app.test.js

```diff
@@ -267,8 +267,28 @@ describe('/api/reviews/:review_id', () => {
 
 });
 
+describe('/api/users', () => {
+    test('GET-Status: 200 - responds with array of all users ', () => {
+        return request(app).get("/api/users").expect(200).then(({ body }) => {
 
 
+            expect(Array.isArray(body.users)).toBe(true)
```

Comment:
As you're using a forEach here - check the length of the user array and expect it to be the correct length.